strings/0002/sort-chars-ascending_francine.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+char *sortAscending(char[]);
+
+int main()
+{
+    char origArray[100];
+    char *newArray;
+    int i, length;
+
+    //ask user to enter string
+    printf("Enter string: ");
+    gets(origArray);
+    length = strlen(origArray);
+
+    //function call
+    newArray = sortAscending(origArray);
+
+    for (i = 0; i < length; i++) {
```

Comment:
I think it'd be better to use `puts` here rather than the loop. It's easier to read.

```c
puts(newArray);
```